assets/styles.css

```diff
@@ -0,0 +1,286 @@
+body {
+  overflow-x: hidden;
+}
+
+.body-wrapper {
+  width: 1210px;
+  margin: 0 auto;
+}
+
+.header-wrapper {
+  height: 100px;
+  color: white;
+  background-color: #f72210;
+}
+
+.font-head a {
+  font-size: 14px;
+}
+
+.logo-head {
+  color: #fff;
+  fill: white;
+  width: 400px;
+}
+
+.header-font {
+  font: bold 13px/24px "Roboto Condensed", sans-serif;
+  font-size: 1rem;
+  font-weight: 400;
+  line-height: 1.5rem;
+}
+
+.rounded-pill {
+  background-color: #ffe400;
+  height: 30px;
+}
+
+.search-input {
+  height: 38px;
+  width: max-content;
+}
+
+.article-body {
+  padding-top: 10px;
+  margin-bottom: 15px;
+}
+
+.main-section {
+  padding: 25px;
+}
+
+.navbar-one a {
+  padding: 15px;
+}
+
+.navbar-one li {
+  border-right: 1px solid #dee2e6 !important;
+}
+
+.newsletter-signup {
+  max-width: 300px;
+  margin: 0 auto 30px;
+  border: 1px solid #e5e5e5;
+  background-color: #f4f5f6;
+  padding: 20px;
+}
+
+.icon-signup {
+  flex: none;
+  display: block;
+  width: 29px;
+  height: 28px;
+  background: url(https://g.newsweek.com/www/images/logo-n1.svg) center/contain no-repeat #f72210;
+  margin-right: 10px;
+}
+
+.free-signup-text {
+  font: bold 19px/19px "Roboto Condensed", sans-serif;
+  text-transform: uppercase;
+}
+
+.login-text-icon {
+  display: block;
+}
+
+.login-text-logo {
+  display: none;
+}
+
+.subscribe-img-top {
+  z-index: 2;
+  margin-left: -150px;
+}
+
+.email-newsletter-img {
+  object-fit: cover;
+  width: 170px;
+  height: 370px;
+}
+
+.w-90 {
+  width: 90% !important;
+}
+
+.category-label {
+  display: block;
+  font: bold 14px "Roboto Condensed", sans-serif;
+  text-transform: uppercase;
+  padding: 6px 10px 6px;
+  background-color: rgba(255, 255, 255, 0.9);
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  z-index: 1;
+}
+
+.category-label > a {
+  color: #f72210 !important;
+}
+
+.article-wrap {
+  position: relative;
+  min-height: 33px;
+  margin-bottom: 10px;
+}
+
+.culture-image-wrap {
+  position: relative;
+  min-height: 33px;
+  margin-bottom: 10px;
+}
+
+.culture-image-tag {
+  font: bold 13px/20px "Roboto condensed", Arial, sans-serif;
+  text-transform: uppercase;
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  display: inline-block;
+  background-color: #c2d4d7 !important;
```

Comment:
- [ ] You are overusing the `!important` tag in your stylesheet.
It's possible to complete this project and properly apply styles without using it so often, it's not a good CSS practice to rely on `!important` instead of specificity.

Please try to reduce as much as possible the use to `!important`, you can do that by using more specific selectors, combining multiple classes, etc.

Reply:
Reviewed and replacing as many as possible.